gis/tilemaker/Portfile

```diff
@@ -0,0 +1,35 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           github 1.0
+PortGroup           cmake 1.0
```

Comment:
please use the latest cmake PG, which is 1.1 - if you do that, then you can also remove the " cmake.out_of_source yes" below as that is now the default